miro_preprocessor/main.tf

```diff
@@ -55,21 +55,42 @@ module "miro_image_sorter" {
   topic_none_publish_policy          = "${module.none_topic.publish_policy}"
 }
 
-module "miro_copy_s3_asset" {
-  source                         = "miro_copy_s3_asset"
-  topic_miro_copy_s3_asset_arn   = "${module.catalogue_api_topic.arn}"
-  topic_miro_image_to_dynamo_arn = "${module.topic_miro_image_to_dynamo.arn}"
+module "miro_copy_catalogue_derivative" {
+  source                        = "miro_copy_s3_asset"
+  topic_miro_copy_s3_asset_arn  = "${module.catalogue_api_topic.arn}"
+  topic_forward_sns_message_arn = "${module.topic_miro_image_to_dynamo.arn}"
+
+  lambda_error_alarm_arn       = "${local.lambda_error_alarm_arn}"
+  bucket_destination_asset_arn = "${local.bucket_miro_images_public_arn}"
+  bucket_destination_name      = "${local.bucket_miro_images_public_name}"
+  bucket_source_asset_arn      = "${local.bucket_miro_images_sync_arn}"
+  bucket_source_asset_name     = "${local.bucket_miro_images_sync_name}"
+
+  lambda_description     = "Copy catalogue miro derivatives to Loris s3 bucket"
+  lambda_name            = "miro_copy_catalogue_derivative"
+  is_master_asset        = "false"
```

Comment:
Terraform supports booleans as a first-class type. Let’s use those, rather than strings.

https://www.terraform.io/docs/configuration/syntax.html

Reply:
Yeah I did try that, but I couldn't get this to work properly with booleans 
```
locals {
  source_prefix  = "${var.is_master_asset == "true" ? "Wellcome_Images_Archive": "fullsize/"}"
  lambda_s3_type = "${var.is_master_asset == "true" ? "master": "derivative"}"
}
```